docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1201,17 +1211,22 @@ quarkus tls lets-encrypt issue-certificate \
 ----
 <1> Set your domain name.
 <2> Provide your contact email address that Let's Encrypt can use to contact you in case of any issues with your Let's Encrypt account.
-<3> Set your application management URL which can be used to handle ACME challenges. Use `https://localhost:8443/` if you chose not to enable a management router in the <<lets-encrypt-prerequisites>> step.
+<3> Set your application management URL, which can be used to handle ACME challenges. Use `https://localhost:8443/` if you choose not to enable a management router in the <<lets-encrypt-prerequisites>> step.
 
-During this command, the TLS registry CLI checks if the application is prepared to serve the challenge, creates and records Let's Encrypt account information, issues a Let's Encrypt certificate request, and interacts with the Quarkus application to resolve ACME challenges.
+During this command, the TLS registry CLI performs the following tasks:
```

Comment:
Style suggestion: Maybe say "During the processing of the `issue-certificate` command ......"